indico_toolkit/snapshots/snapshot.py

```diff
@@ -252,4 +287,4 @@ def __repr__(self):
 
     @property
     def number_of_samples(self) -> int:
-        return self.df.shape[0]
+        return self.df.shape[0]
```

Comment:
Make sure to run `black` formatting on files before committing. We can help you with that if you haven't used it before, there's utilities for it in VS Code.